source/common/mongo/proxy.h

```diff
@@ -73,6 +77,31 @@ class AccessLog {
 typedef std::shared_ptr<AccessLog> AccessLogSharedPtr;
 
 /**
+ * Mongo fault configuration.
+ */
+class FaultConfig {
```

Comment:
this is to abstract the way FaultConfig is constructed, so to have a handy (simplistic) implementation for the test.
This provides additional flexibility for testing purposes.